modules/backend/traits/UploadableWidget.php

```diff
@@ -171,7 +176,7 @@ public function validateMediaFileName(string $fileName, string $extension): stri
         /*
          * File name contains non-latin characters, attempt to slug the value
          */
-        if (!$this->validateFileName($fileName)) {
+        if (!$this->validateFileName($fileName) && $this->validateMediaFileName) {
```

Comment:
Why would we want to allow a filename to skip validation? The validation exists for a reason.

Reply:
When I'm adding multiple files and their have name formatting that i use elsewhere in a plugin so they can be associated with a relation for upload after batch uploading (An argument can be made to use fileupload, but the batch uploading is for individual relations that use file upload not just one using multiupload). I made it opt-out, but I do need it to keep the name formatting.

Reply:
I wouldn't recommend abusing the media library system for that. I'd need to know more about your exact use case, but I could see you using an attachMany to accept multiple files and then re-assigning them to the relevant attachOne relationships post upload. Also why can't you have the files renamed when they get moved out of the medialibrary?

Reply:
I decided to create a clone of the mediamanager as a plugin/widget rather than using that media manager.